main.go

```diff
@@ -403,7 +405,7 @@ func main() {
 		fmt.Printf("# sending %d %s req/s with concurrency=%d using url list %s ...\n", (*qps * *concurrency), *method, *concurrency, urldest[1:])
 	}
 
-	fmt.Printf("# %s good/b/f t   goal%% %s min [p50 p95 p99  p999]  max bhash change\n", timePadding, intPadding)
+	fmt.Printf("# %s iteration good/b/f t   goal%% %s min [p50 p95 p99  p999]  max bhash change\n", timePadding, intPadding)
```

Comment:
In order to keep the alignment correct for the default case, would you mind adding a few more formatting changes...

```diff
diff --git a/main.go b/main.go
index 1091ad5..90b225f 100644
--- a/main.go
+++ b/main.go
@@ -395,7 +395,7 @@ func main() {
 	var sendTraffic sync.WaitGroup
 	// The time portion of the header can change due to timezone.
 	timeLen := len(time.Now().Format(time.RFC3339))
-	timePadding := strings.Repeat(" ", timeLen)
+	timePadding := strings.Repeat(" ", timeLen-len("# "))
 	intLen := len(fmt.Sprintf("%s", *interval))
 	intPadding := strings.Repeat(" ", intLen-2)
 
@@ -405,7 +405,7 @@ func main() {
 		fmt.Printf("# sending %d %s req/s with concurrency=%d using url list %s ...\n", (*qps * *concurrency), *method, *concurrency, urldest[1:])
 	}
 
-	fmt.Printf("# %s iteration good/b/f t   goal%% %s min [p50 p95 p99  p999]  max bhash change\n", timePadding, intPadding)
+	fmt.Printf("# %s iter   good/b/f t   goal%% %s min [p50 p95 p99  p999]  max bhash change\n", timePadding, intPadding)
 	stride := *concurrency
 	if stride > len(dstURLs) {
 		stride = 1
@@ -494,7 +494,7 @@ func main() {
 			changeIndicator := window.CalculateChangeIndicator(latencyHistory.Items, lastP99)
 			latencyHistory.Push(lastP99)
 
-			fmt.Printf("%s %d %6d/%1d/%1d %d %3d%% %s %3d [%3d %3d %3d %4d ] %4d %6d %s\n",
+			fmt.Printf("%s %4d %6d/%1d/%1d %d %3d%% %s %3d [%3d %3d %3d %4d ] %4d %6d %s\n",
 				t.Format(time.RFC3339),
 				iteration,
 				good,
```

For reference, here's the existing output on master:
```
# sending 1 GET req/s with concurrency=1 to http://localhost:4140/ ...
#                           good/b/f t   goal%   min [p50 p95 p99  p999]  max bhash change
2018-10-08T09:43:31-07:00      9/0/0 10  90% 10s  11 [ 13  70  70   70 ]   70      0 +
2018-10-08T09:43:41-07:00     10/0/0 10 100% 10s  11 [ 12  15  15   15 ]   15      0
2018-10-08T09:43:51-07:00     10/0/0 10 100% 10s  10 [ 13  15  15   15 ]   15      0
```
...and here's the output with the above diff applie:
```
# sending 1 GET req/s with concurrency=1 to http://localhost:4140/ ...
#                         iter   good/b/f t   goal%   min [p50 p95 p99  p999]  max bhash change
2018-10-08T10:23:14-07:00    0      9/0/0 10  90% 10s  10 [ 11  13  13   13 ]   13      0 +
2018-10-08T10:23:24-07:00    1     10/0/0 10 100% 10s  11 [ 11  14  14   14 ]   14      0
```